packages/providers/src/provider.test.ts

```diff
@@ -197,4 +198,46 @@ describe('Provider', () => {
     const provider = new Provider(providerUrl, { fetch: customFetch });
     expect(await provider.getVersion()).toEqual('0.30.0');
   });
+
+  it('can produce blocks with custom timestamps', async () => {
+    const provider = new Provider('http://127.0.0.1:4000/graphql');
```

Comment:
@camsjams was looking into this and just realized this is because the timestamp for the latest block returned by the GraphQL API is 'invalid': 
<img width="1318" alt="image" src="https://user-images.githubusercontent.com/39617427/231410625-f6de1037-d4e1-49f1-97ba-15dde8c7ece1.png">

That doesn't look like a valid UNIX timestamp. I've messaged the client channel about this.

Reply:
Turns out the client and the GraphQL API return a `tai64` timestamp instead of UNIX timestamps now. Hence the confusion.

Reply:
I'm still researching for a reliable method to convert `tai64` timestamps into `UNIX`, so that we can have a built-in robust method for this conversion in our SDK.